firebase-perf/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 
 # 21.0.2
+* [fixed] Fixed a performance issue with shared preferences
+  calling `.apply()` every time a value is read from remote config (#6407)
```

Comment:
isn't `21.0.2` the latest released version?
https://firebase.google.com/support/release-notes/android

Reply:
The 21.0.2 is released in `Oct 21, 2024` based on here: https://mvnrepository.com/artifact/com.google.firebase/firebase-perf/21.0.2